certified-connectors/My Acclaro/apiDefinition.swagger.json

```diff
@@ -518,7 +518,7 @@
         "tags": [
           "file"
         ],
-        "description": "Deletes the given file from the order, prior to an order being submitted. If a source file ever needs to be cancelled after it’s been submitted as part of an order, ‘CANCELL FILE’ can be submitted as a comment in /orders/{orderid}/files/{fileid}/comment. We also request that you separately contact your Acclaro project manager with the source file cancellation request.",
+        "description": "Deletes the given file from the order, prior to an order being submitted. If a source file ever needs to be CANCELed after it’s been submitted as part of an order, ‘CANCEL FILE’ can be submitted as a comment in /orders/{orderid}/files/{fileid}/comment. We also request that you separately contact your Acclaro project manager with the source file CANCELation request.",
```

Comment:
@msanz-acclaro please fix the casing for descriptions, e.g. `canceled`, `cancelation`.

Reply:
@v-EgorKozhadei I went `sed --ignore-case...` oh god, one sec...

Reply:
corrected, please don't tell my boss :-P

Reply:
@msanz-acclaro there's one more word at the very end of this string.

Reply:
@msanz-acclaro could you please fix the one at the end of the string too? :)

Reply:
yes, sorry, just not my day. Tried to do a fast fix and messed it up more...

Reply:
@msanz-acclaro everything is OK, thank you very much for an update!